realm/realm-library/src/objectServer/java/io/realm/SyncSession.java

```diff
@@ -324,7 +324,8 @@ protected void onSuccess(AuthenticateResponse response) {
             @Override
             protected void onError(AuthenticateResponse response) {
                 onGoingAccessTokenQuery.set(false);
-                RealmLog.debug("Session[%s]: Failed to get access token (%d)", configuration.getPath(), response.getError().getErrorCode());
+                RealmLog.debug("Session[%s]: Failed to get access token (%s)", configuration.getPath(),
+                        response.getError().getErrorCode());
```

Comment:
`String.format()` will do it automatically. 

 I think we should not call `toString()` since that call is always executed if we fixed #4734.